### ISSUE TITLE
[API docs] Modules should return an array of specification files

### ISSUE THREAD

**Issue by xlecours** [aces/Loris#7749]
For instance, the api module has a schema and a schema-{version}-dev file.
Being able  to switch version would be nice.

**Comment by driusan** [comment]
I agree.. I think I said this on the initial PR but we decided to merge it to expediate it..

**Comment by xlecours** [comment]
> I agree.. I think I said this on the initial PR but we decided to merge it to expediate it..

True :)

**Comment by xlecours** [comment]
This is still an issue
